src/FsToolkit.ErrorHandling.IcedTasks/CancellableTaskValidationCE.fs

```diff
@@ -0,0 +1,169 @@
+﻿namespace FsToolkit.ErrorHandling
+
+open System
+open IcedTasks
+
+[<AutoOpen>]
+module CancellableTaskValidationCE =
```

Comment:
Given the criteria for supporting various other binds (described in another comment), this may end up having to become a resumable CE like it's Result counterpart.